lib/portlayer/exec/container.go

```diff
@@ -377,15 +382,23 @@ func (c *Container) Signal(ctx context.Context, num int64) error {
 	return c.startGuestProgram(ctx, "kill", fmt.Sprintf("%d", num))
 }
 
-func (c *Container) LogReader(ctx context.Context) (io.ReadCloser, error) {
+func (c *Container) stopped() {
```

Comment:
eventCallback will only execute onStop if the event is stopping the container -- so out of band stops / power offs, etc...or a short lived container that exits and powers down.  Executing a client side stop will not.  

So thinking potentially others might call onStop and didn't want to recreate the if statement in those places...no biggie...so whatever you feel best about